### ISSUE TITLE
Website Enhancement Project-4-Make expansible menu in the API documentation not clickable

### ISSUE THREAD

**Issue by shanyi15** [PaddlePaddle/PaddlePaddle.org#382]
**Background**
优化PaddlePaddle.org，提升用户使用体验。本次我们主要focus on V2版本。

**How to**
我们拆分出以下官网优化方案：
https://shimo.im/sheet/uoMu6uOR6sE3Mj5L/
我们将按照序号分别提issue，需要参考的MRD链接已在上述文档中给出

**优化官网方案-4-优化左侧导航栏**
左侧导航栏中，部分标题，设置成”只能展开，不能点击“。

举例如图：

![image](https://user-images.githubusercontent.com/35982308/35615647-bfbcb526-06ad-11e8-888f-25a1d45da3e7.png)

**Comment by cs2be** [comment]
Fixed in https://github.com/PaddlePaddle/PaddlePaddle.org/pull/391

**Comment by shanyi15** [comment]
Hi, I see the changes on paddlepaddle.org. ：）

Please note that "not clickable" is only suitable for [API ](http://www.paddlepaddle.org/docs/develop/api/en/v2/config/activation.html), however, not suitable for [Documentations](http://www.paddlepaddle.org/docs/develop/documentation/zh/getstarted/quickstart_cn.html).
For example, after clicking "新手入门", there will be [a Guide Page ](http://www.paddlepaddle.org/docs/develop/documentation/zh/getstarted/index_cn.html)(or we can call it 新手入门‘s mainpage) and will not directly jump to ["快速开始"](http://www.paddlepaddle.org/docs/develop/documentation/zh/getstarted/quickstart_cn.html)
Since we have started [Guide Writing Work](https://github.com/PaddlePaddle/Paddle/issues?q=is%3Aissue+is%3Aopen+label%3Adocumentation+author%3Ashanyi15) yesterday, it's very necessary for us to keep  Guide Page.

The effect of "not clickable" could be like [pytorch doc](http://pytorch.org/docs/0.3.0/)
"NOTES" "PACKAGE REFERENCES" "TORCHVISION REFERENCE", there is no hyperlink, the buttons cannot be clicked.

@cs2be Thank you so much!

**Comment by cs2be** [comment]
i see, however some parent links will just show a menu of the child links, which is something luo tao mention she didn’t want.  can you confirm this with her?

**Comment by shanyi15** [comment]
sure, @luotao1  and i have discussed, our solution is adding some content on parent page. so it will be like this:
(take [tensorflow](https://www.tensorflow.org/programmers_guide/) for example)
![image](https://user-images.githubusercontent.com/35982308/36013095-3122b846-0d9d-11e8-8607-253a4f4b8c64.png)

and the guide writing work is a task that ask our RD to write something on this page.
we just started writing guide yesterday, this work will be finished in early next month, 
therefore we can keep the parent page show, and after the guide writing work is finished, 
this page will look like tensorflow guide page.

: )
Thank you

**Comment by cs2be** [comment]
Sure, so I can add back in the top level links for Documentation, and remove it for API?

On Feb 8, 2018, at 9:40 PM, Shan Yi <notifications@github.com<mailto:notifications@github.com>> wrote:


sure, @luotao1<https://github.com/luotao1> and i have discussed, our solution is adding some content on parent page. so it will be like this:
(take tensorflow<https://www.tensorflow.org/programmers_guide/> for example)
[image]<https://user-images.githubusercontent.com/35982308/36013095-3122b846-0d9d-11e8-8607-253a4f4b8c64.png>

and the guide writing work is a task that ask our RD to write something on this page.
we just started writing guide yesterday, this work will be finished in early next month,
therefore we can keep the parent page show, and after the guide writing work is finished,
this page will look like tensorflow guide page.

: )
Thank you

**Comment by shanyi15** [comment]
Yes, and there is one thing needs to be mentioned, after clicking "API"  on the top navigation. The page will jump to an API's main page, please see https://github.com/PaddlePaddle/PaddlePaddle.org/issues/383 (sorry that this issue is written in Chinese, i'll translate it as soon as possible) and the content will be adding on this page is in https://github.com/PaddlePaddle/Paddle/issues/8042

In sum, please adding an API main page, and link it to "API" on the top navigation.

About the left navigation in "API" section, it will be like the following image. and the third level links will be put in the left navigation. Please see #384 (I'll translate ASAP)
![image](https://user-images.githubusercontent.com/35982308/36013942-1ffb42fe-0da2-11e8-9da0-3f1d7f49b2aa.png)

however, in Documentation Section, there is a special menu, it will show the 4th level link in the left navigation ,please see: https://github.com/PaddlePaddle/PaddlePaddle.org/issues/392

**Comment by shanyi15** [comment]
Plus: add back in the top level links for Documentation, and remove it for API is right~ detail is drawn in the last comment. After click "Documentation" on the top navigation, it will jump to "Get Started(新手入门)" page.

**Comment by shanyi15** [comment]
The 2 issues mentioned above have been translated. : )

**Comment by cs2be** [comment]
Fixed in https://github.com/PaddlePaddle/PaddlePaddle.org/pull/401
